README.md

```
@@ -1,73 +1,59 @@
 
-## 🚀 Getting Started
+🚀 Getting Started
 
 Follow these steps to set up the development environment and test the contracts locally.
 
-### 1. Clone the Repository
-
-```bash
-git clone https://github.com/KiiChain/kii-solidity-contracts
+1. Clone the Repository
+git clone https://github.com/KiiChain/kii-solidity-contracts.git
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)